app/views/contactPreference/EmailPreferenceView.scala.html

```diff
@@ -44,7 +44,7 @@
           no  -> messages("common.no")),
         question = messages("emailPreference.title"),
         additionalContent = Some(questionContent),
-        inline = true
+        inline = false
```

Comment:
Minor bug found - radio buttons are meant to be vertical not horizontal so updating to be consistent